internal/kafka/internal/migrations/20221219154830_add_kafka_promote_worker_in_leader_leases.go

```diff
@@ -0,0 +1,26 @@
+package migrations
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/db"
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+var leaderLeaseType = "promoting_kafka"
```

Comment:
Can we make it a local variable of the addKafkaPromoteWorkerInLeaderLeases method to avoid polluting the package-level namespace?